src/overlays/ovl_i2/fox_me.c

```diff
@@ -1,97 +1,2527 @@
+    Actor_Initialize(actor1);
+    actor1->obj.status = 1;
+    actor1->obj.id = OBJ_ACTOR_182;
+
+    actor1->obj.pos.x = actor2->obj.pos.x + x;
+    actor1->obj.pos.y = actor2->obj.pos.y + y;
+    actor1->obj.pos.z = actor2->obj.pos.x + z;
+
+    actor1->obj.rot.y = Rand_ZeroOneSeeded() * 360.0f;
+    actor1->obj.rot.x = Rand_ZeroOneSeeded() * 360.0f;
+
+    actor1->timer_0C2 = 0x2710;
+    actor1->vel.z = 30.0f;
+    Object_SetInfo(&actor1->info, actor1->obj.id);
+}
+
+void func_i2_8018CAD8(void) {
+    Actor* actor = &gActors[50];
+
+    Actor_Initialize(actor);
+    actor->obj.status = 1;
+    actor->obj.pos.x = 0.0f;
+    actor->obj.pos.y = 0.0f;
+    actor->obj.pos.z = 9000.0f;
+    actor->unk_0B6 = 10;
+    actor->obj.id = 0xC3;
+    Object_SetInfo(&actor->info, actor->obj.id);
+}
+
+void func_i2_8018CB50(Effect* effect, Actor* actor) {
+    Effect_Initialize(effect);
+    effect->obj.status = 2;
+    effect->obj.id = OBJ_EFFECT_346;
+    effect->timer_50 = (s32) (Rand_ZeroOne() * 20.0f) + 20.0f;
+    effect->scale2 = (Rand_ZeroOne() * 0.5f) + 0.5f;
+
+    effect->obj.pos.x = actor->obj.pos.x;
+    effect->obj.pos.y = actor->obj.pos.y;
+    effect->obj.pos.z = actor->obj.pos.z;
+
+    effect->vel.x = (Rand_ZeroOne() - 0.5f) * 30.0f;
+    effect->vel.y = (Rand_ZeroOne() - 0.5f) * 30.0f;
+    effect->vel.z = (Rand_ZeroOne() - 0.5f) * 30.0f;
+
+    effect->obj.rot.z = Rand_ZeroOne() * 360.0f;
+    Object_SetInfo(&effect->info, effect->obj.id);
+}
+
+void func_i2_8018CCF8(Actor* arg0) {
+    s32 i;
+    s32 j;
+
+    func_8007A6F0(&arg0->obj.pos, 0x2903A008);
+
+    for (i = 0; i < 25; ++i) {
+        for (j = 0; j < ARRAY_COUNT(gEffects); j++) {
+            if (gEffects[j].obj.status == 0) {
+                func_i2_8018CB50(&gEffects[j], arg0);
+                break;
+            }
+        }
+    }
+}
+
+void func_i2_8018CD8C(Player* player) {
+    u8 sp8F;
+    s32 i;
+    Actor* actor0 = &gActors[0];
+    Actor* actor1 = &gActors[1];
+    Actor* actor2 = &gActors[2];
+    Actor* actor3 = &gActors[3];
+    f32 x;
+    f32 y;
+    f32 z;
+    f32 sp68;
+    f32 sp64;
+    Vec3f sp58;
+    Vec3f sp4C;
+
+    D_8017835C = 4;
+
+    switch (player->unk_1D0) {
+        case 0:
+            D_80177A80 = 0;
+
+            if (gTeamShields[1] > 0) {
+                func_i2_8018C77C(actor0, 0);
+            }
+            if (gTeamShields[2] > 0) {
+                func_i2_8018C77C(actor1, 1);
+            }
+            if (gTeamShields[3] > 0) {
+                func_i2_8018C77C(actor2, 2);
+            }
+
+            func_i2_8018C77C(actor3, 3);
+            func_i2_8018CAD8();
+
+            for (i = 5; i < 15; i++) {
+                func_i2_8018C8F4(&gActors[i], actor3);
+            }
+
+            D_80177A48[1] = -13000.0f;
+            D_80177A48[2] = -13000.0f;
+            D_80177A48[4] = -22000.0f;
+
+            player->camEye.x = (D_80177978 = player->pos.x + 100.0f);
```

Comment:
```suggestion
            player->camEye.x = D_80177978 = player->pos.x + 100.0f;
```
I don't think parentheses are needed in chain assigns. In fact I find them distracting